src/problem3.rb

```diff
@@ -0,0 +1,12 @@
+class Problem3
+  def contains_duplicates(array)
+    return false if array.nil? or array.empty?
+
+    array_length_sum = 0;
+    (0..array.length).each do |n|
```

Comment:
:) the c# in you is strong, ruby has intervals, one can write`(0..array.lenght).sum`